int16[6] velocity_data
int16[6] current_data
int32[6] position_data
